### ISSUE TITLE
JSON Linked Data, Schemas and Global Market Extensions (the What, the How and the Who)

### ISSUE THREAD

**Issue by PatStLouis** [uncefact/spec-untp#122]
The UNTP has a requirement to make data meaningful to international parties across industries. To do this, we need to add semantic meaning to this data which can be commonly understood (minimize ambiguity, the what), provide a strong extensible schematic (having a data model, the how) and add proof of provenance (signatures attesting a known entity had made these claims, the who). All this while maintaining the adoption barrier as low as possible.

These 3 points are addressed with Linked Data, Schemas and Digital Signatures respectively.

This is why the Verifiable Credential Data Model (VCDM) was selected as a suitable vehicle of information.

When we defined the CORE VC types of the UNTP:
- DigitalProductPassport (DPP)
- DigitalConformityCredential (DCC)
- DigitalTraceabilityEvent (DTE)
- DigitalIdentityAnchor (DIA)

We also make sure that these can be extended to meet specific regulatory requirements from participants, whether they are issuers of information or consumers of information. These extensions are primarily driven be adding new semantic information through additional properties and types. And this new information needs to be added to the existing schemas, creating a new schema which is unique to it's extension (the accumulation of the core elements and the extended elements).

Making this manageable can pose a significant challenge, as semantic data is conveyed through context urls whilst schematic data is conveyed through a json schema. 

The UNTP currently leverages the 2 following VCDM features:
- evidence
- credentialStatus

I would like to propose the [credentialSchema](https://www.w3.org/TR/vc-data-model-2.0/#data-schemas) as a means to provide schematic information.

An extension would provide the core vc type context url in conjunction with an additional context url defining terms part of the extension.

This same extension would also provide json schema urls.

There are 2 ways to achieve this, [JsonSchema](https://www.w3.org/TR/vc-json-schema/#jsonschema) and [JsonSchemaCredential](https://www.w3.org/TR/vc-json-schema/#jsonschemacredential).

Part of tackling this issue would be to evaluated which one to recommend and see if this is fit for this purpose.

**Comment by nissimsan** [comment]
Agreed. I had this included this on my example too: https://github.com/uncefact/spec-untp/blob/main/website/samples/untp-digital-product-passport.json#L7
